duckduckgo/iOS
# Subscription Auth Fixes

Task/Issue URL: https://app.asana.com/0/0/1207023339673018/f

**Description**:

**Steps to test this PR**:
1.  Subscribe to Prvacy Pro
2. Wait 20m
3. Open app and turn on wifi mode
4. Open settings
5. Subscription should be visible in settings




---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
